scipy/linalg/_basic.py

```diff
@@ -970,21 +982,13 @@ def det(a, overwrite_a=False, check_finite=True):
     """
     Compute the determinant of a matrix
 
-    The determinant of a square matrix is a value derived arithmetically
-    from the coefficients of the matrix.
-
-    The determinant for a 3x3 matrix, for example, is computed as follows::
-
-        a    b    c
-        d    e    f = A
-        g    h    i
-
-        det(A) = a*e*i + b*f*g + c*d*h - c*e*g - b*d*i - a*f*h
+    The determinant is a scalar that is a function of the associated square
```

Comment:
Much better. ("**the** scalar" was the main issue here before, unless we were going to define what unique scalar we were referring to.)